mevolib/inference/__init__.py

```diff
@@ -51,19 +68,19 @@
 
 #-------------------------------------------------------------------------------
 
-def get_tools ( ) :
+def get_tools() -> dict:
```

Comment:
```suggestion
def get_tools() -> dict[str, list]:
```
Adding a little bit more information makes it easier to detect bugs.